packages/fx-core/src/core/middleware/localSettingsLoader.ts

```diff
@@ -10,13 +10,14 @@ import { PluginNames } from "../../plugins/solution/fx-solution/constants";
 import { getActivatedResourcePlugins } from "../../plugins/solution/fx-solution/ResourcePluginContainer";
 import { ObjectIsUndefinedError } from "../error";
 import { shouldIgnored } from "./projectSettingsLoader";
-import { IsSimpleAuthEnabled } from "../../common/tools";
+import { isConfigUnifyEnabled, IsSimpleAuthEnabled } from "../../common/tools";
+import { isPureExistingApp } from "../utils";
```

Comment:
What's this for?

Reply:
It comes from merging code; I will delete it. Thanks